packages/plans-grid/src/plans-details/index.tsx

```diff
@@ -51,6 +51,18 @@ const PlansDetails: React.FunctionComponent< Props > = ( { onSelect, locale, bil
 	const isLoading = ! supportedPlans?.length;
 	const placeholderPlans = [ 1, 2, 3, 4, 5 ];
 
+	const newAnnualBillingLabel = __( 'Monthly price (billed yearly)', __i18n_text_domain__ );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Monthly subscription (billed yearly)' )` *ES Score: 9*